pgmpy/estimators/PC.py

```diff
@@ -392,7 +418,43 @@ def _parallel_fun(u, v):
 
         if show_progress and config.SHOW_PROGRESS:
             pbar.close()
-        return graph, separating_sets
+        return graph, separating_sets, temporal_ordering
+
+    @staticmethod
+    def _get_temporal_ordering(temporal_order):
+        ordering = dict()
+        for order, tier in enumerate(temporal_order):
+            for node in tier:
+                ordering[node] = order
+
+        return ordering
+
+    @staticmethod
+    def _get_temporal_separating_set(u, v, temporal_ordering, temporal_order):
+        if temporal_ordering == {}:
+            return set()
+
+        max_order = min(temporal_ordering[u], temporal_ordering[v])
+        overall_max_order = max(temporal_ordering.items())
+        separating_set = set()
+        for tier in range(max_order + 1, overall_max_order + 1):
+            separating_set = separating_set.union(set(temporal_order[tier]))
+
+        separating_set.discard(u)
+        separating_set.discard(v)
+
+        return separating_set
+
+    @staticmethod
+    def _orient_temporal_forbidden_edges(skel, temporal_ordering):
```

Comment:
Can we move it to ExpertKnowledge as well and directly modify the ExpertKnowledge.forbidden_edges ?